src/index.js

```diff
@@ -0,0 +1,146 @@
+const state = {
+  temp: 50,
+  city: 'Bloomington',
+  lat: 39.1670396,
+  lon: -86.5342881,
+  sky: 'sunny',
+};
+
+const upButton = document.getElementById('up-arrow');
+const downButton = document.getElementById('down-arrow');
+const tempText = document.getElementById('current-temp');
+const ground = document.getElementById('ground');
+const cityHeader = document.getElementById('city-name');
+const cityInput = document.getElementById('city');
+const realTemp = document.getElementById('realtime-temp');
+const skySelect = document.getElementById('sky-select');
+const skyLine = document.getElementById('sky');
+const landscape = document.getElementById('landscape');
+const cityReset = document.getElementById('city-reset');
+
+const setTempColor = (temp) => {
+  if (temp >= 80) {
+    tempText.style.color = 'red';
+  } else if (temp >= 70) {
+    tempText.style.color = 'orange';
+  } else if (temp >= 60) {
+    tempText.style.color = 'yellow';
+  } else if (temp >= 50) {
+    tempText.style.color = 'green';
+  } else {
+    tempText.style.color = 'teal';
+  }
+};
```

Comment:
Check out the logic in setTempColor() and how similar the branches are of the conditional statement. We look for whether the temperature is within a range, and pick a color accordingly. What if we had a list of records that we could iterate through to find the values. We could set up something like

```javascript
const TEMP_COLORS = [ 
  { upperBound: 49, color: "teal"}, 
  { upperBound: 59, color: "green"}, 
  { upperBound: 69, color: "yellow"}, 
  { upperBound: 79, color: "orange"}, 
  { upperBound: 80, color: "red"}
];
```

Then your method would iterate through `TEMP_COLORS` and find the first record that has an upper bound higher than our temperature, then use it as the source of picking the color which would help make the function a bit more concise.